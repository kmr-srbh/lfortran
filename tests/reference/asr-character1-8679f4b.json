{
    "basename": "asr-character1-8679f4b",
    "cmd": "lfortran --show-asr --implicit-interface --no-color {infile} -o {outfile}",
    "infile": "tests/character1.f90",
    "infile_hash": "83f86b3c6ec94d478c82439a20e7af4238dbd1d867cae8efbb7b1f39",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-character1-8679f4b.stdout",
    "stdout_hash": "f119895b9eaf31097bb11e2710d08dd96859abc3d0eb502ef7941362",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}