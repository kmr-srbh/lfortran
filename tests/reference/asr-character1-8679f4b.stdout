(TranslationUnit
    (SymbolTable
        1
        {
            xerror:
                (Function
                    (SymbolTable
                        2
                        {
                            mess:
                                (Variable
                                    2
                                    mess
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Character 1 1 () PointerString)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    xerror
                    (FunctionType
                        [(Character 1 1 () PointerString)]
                        ()
                        Source
                        Implementation
                        ()
                        .false.
                        .false.
                        .false.
                        .false.
                        .false.
                        []
                        .false.
                    )
                    []
                    [(Var 2 mess)]
                    [(Print
                        [(StringItem
                            (Var 2 mess)
                            (IntegerConstant 1 (Integer 4) Decimal)
                            (Character 1 1 () PointerString)
                            ()
                        )]
                        ()
                        ()
                    )
                    (Return)]
                    ()
                    Public
                    .false.
                    .false.
                    ()
                )
        })
    []
)
